cloudbuild_CI.yaml

```diff
@@ -55,4 +55,4 @@ steps:
       # - '--gs_dir bashir-variant_integration_test_runs'
 images:
   - 'gcr.io/${PROJECT_ID}/gcp-variant-transforms:${COMMIT_SHA}'
-timeout: 45m
+timeout: 60m
```

Comment:
Unfortunately, these two integration tests do not run in parallel.

Reply:
Hmm, can you elaborate why they are not run in parallel? Aren't we using TestRunner.run() in both cases and that is supposed to submit all requests and let them run together?

Reply:
Oh, now I realized what you mean by "these two integration tests" in your original comment; I thought you are talking about the two tests you added for preprocessor. Okay, I think this is fine for now.